.ci/azure-pipelines.yml

```diff
@@ -0,0 +1,98 @@
+trigger:
+  branches:
+    include:
+    - refs/tags/*
```

Comment:
I like the CI to trigger on everything so we can validate PRs and feature branches as well.

Reply:
Triggering on certain/all branches would be possible, the same goes for PRs. However, I doubt it would be the intention to create a Github Release every time CI is triggered. 

Depending on the goal, we could release an artifact for every build, or just check whether or not the build passes (maybe include some tests later on).

Reply:
What I do everywhere else is generate unsigned builds and push them to Azure for every commit, and then sign them and upload to GitHub and the repository if there's a tag.

Reply:
Done!